src/components/SingleRev.jsx

```diff
@@ -22,8 +23,12 @@ export const SingleRev = () => {
   const [voted, setVoted] = useState(false);
   const [deletingComm, setDeletingComm] = useState(false);
```

Comment:
Is this state needed? Assumed it was for showing a message mid-delete but doesn't appear to be used other than in useEffects